homeassistant/components/tado/const.py

```diff
@@ -32,6 +32,19 @@
     PRESET_AWAY,
     PRESET_HOME,
 )
+from homeassistant.components.weather import (
```

Comment:
Looks left over?

Reply:
@bdraco no, it's not.
I'm using this to transform the condition reported by Tado to a home assistant standard value.
See:
https://github.com/home-assistant/core/pull/44807/files#diff-851f046c721e3d1e7fa5b0f9948981858b92bf6e623f23d923b7b6c7add24a98R51-R56
https://github.com/home-assistant/core/pull/44807/files#diff-851f046c721e3d1e7fa5b0f9948981858b92bf6e623f23d923b7b6c7add24a98R185-R191

Reply:
While it does work, and I'm not specifically opposed to this, I'm not sure we should be partially implementing the weather platform on the sensor platform.  I'll ask for a second opinion on this.

Reply:
No we shouldn't use weather constants for the sensor platform.